site-src/blog/2023/0829-welcome-mesh.md

```diff
@@ -0,0 +1,207 @@
+---
+description: >
+  We are excited to announce the v0.8.0 release of Gateway API, where the GAMMA
+  initiative has now reached Experimental status, Gateway and HTTPRoute gain
+  some new capabilities, conformance profiles are supported, and more!
+---
+
+# Gateway API: Welcome, Service Mesh Users!
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: August 29, 2023 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We are thrilled to announce the v0.8.0 release of Gateway API! With this
+release, Gateway API support for service mesh has reached [Experimental
+status][status], and the Gateway and HTTPRoute resources gain some new
+capabilities. We look forward to your feedback!
+
+We're especially delighted to announce that Kuma 2.3+, Linkerd 2.14+, and
+Istio 1.16+ are all fully-conformant implementations of the Gateway API
+service mesh support.
+
+## Why add Service Mesh support to Gateway API?
+
+While the initial focus of Gateway API was always ingress (north-south)
+traffic, it was clear almost from the beginning that the same basic routing
+concepts should also be applicable to service mesh (east-west) traffic. In
+2022, the Gateway API subproject started the [GAMMA initiative][gamma], a
+dedicated vendor-neutral workstream, specifically to examine how best to fit
+service mesh support into the framework of the Gateway API resources, without
+requiring users of Gateway API to relearn everything they understand about the
+API.
+
+Over the last year, GAMMA has dug deeply into the challenges and possible
+solutions around using the Gateway API for service mesh. The end result is a
+small number of [enhancement
+proposals](https://gateway-api.sigs.k8s.io/v1beta1/contributing/gep/) that
+subsume many hours of thought and debate, and provide a minimum viable path to
+allow the Gateway API to be used for service mesh.
+
+### How will mesh routing work when using the Gateway API?
+
+You can find all the details in the [Gateway API Mesh routing
+documentation][mesh-routing] and [GEP-1426], but the short version for Gateway
+API v0.8.0 is that an HTTPRoute can now have a `parentRef` that is a Service,
+rather than just a Gateway. We anticipate future GEPs in this area as we gain
+more experience with service mesh use cases -- binding to a Service makes it
+possible to use the Gateway API with a service mesh, but there are several
+interesting use cases that remain difficult to cover.
+
+As an example, you might use an HTTPRoute to do an A-B test in the mesh as
+follows:
+
+```yaml
+  apiVersion: gateway.networking.k8s.io/v1beta1
+  kind: HTTPRoute
+  metadata:
+    name: bar-route
+  spec:
+    parentRefs:
+      - group: ""
+        kind: Service
+        name: demo-app
+        port: 5000
```

Comment:
Nit: less indent to be consistent with the rest of the fields
```suggestion
    - group: ""
      kind: Service
      name: demo-app
      port: 5000
```

Reply:
Or I guess instead you could indent the following ones like this section 😃

Reply:
😆 Fixed, thanks! 🙂